src/test/java/com/zufar/onlinestore/UserDtoConverterTest.java

```diff
@@ -0,0 +1,90 @@
+package com.zufar.onlinestore;
+
+import com.zufar.onlinestore.user.converter.UserDtoConverter;
+import com.zufar.onlinestore.user.dto.UserDto;
+import com.zufar.onlinestore.user.entity.UserEntity;
+import org.instancio.Instancio;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static io.smallrye.common.constraint.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+class UserDtoConverterTest {
+
+    @Autowired
+    UserDtoConverter converter;
+    private UserEntity user;
+    private UserDto userDto;
+
+    @BeforeEach
+    void setUp() {
+        user = Instancio.create(UserEntity.class);
+        userDto = Instancio.create(UserDto.class);
+    }
+
+    @Test
+    void shouldCheckUserDoesNotEqualToNull() {
+        assertNotNull(user);
```

Comment:
Have you checked Instancio?

Reply:
yes, this case is checking that entity doesn't equal to null